deployment-examples/gcp/README.md

```diff
@@ -1,21 +1,29 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  
+<!-- *generated with [DocToc](https://github.com/thlorenz/doctoc)* -->
+
+- [Continuous Deployment Guide](#continuous-deployment-guide)
+  - [Prerequisites:](#prerequisites)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
 ## Continuous Deployment Guide
 
 As a quick start, you can utilize the available bash script which is capable of deploying both modules to a 
-Google Cloud Project. It requires to have:
+public cloud provider.
```

Comment:
it's not generic, so specify clearly that this guide only makes sense for GCP.

Reply:
e41146b384d6cb54e2b6dda625b946ffc8c92558